src/panels/config/script/ha-script-editor.ts

```diff
@@ -464,9 +409,18 @@ export class HaScriptEditor extends SubscribeMixin(
                           .disabled=${this._readOnly}
                           .dirty=${this._dirty}
                           .saving=${this._saving}
-                          @value-changed=${this._valueChanged}
+                          .errors=${this._errors}
+                          .stateObj=${this._entityId
+                            ? this.hass.states[this._entityId]
+                            : undefined}
+                          .hasBlueprintConfig=${Boolean(this._blueprintConfig)}
+                          .validationErrors=${this._validationErrors}
+                          @duplicate-script=${this._duplicate}
                           @editor-save=${this._handleSaveScript}
+                          @revert-blueprint=${this._revertBlueprint}
                           @save-script=${this._handleSaveScript}
+                          @take-control-save=${this._takeControlSave}
+                          @value-changed=${this._valueChanged}
```

Comment:
I wonder if we shouldn't provide everything to the manual-editor but keep handling it in the editor and add a alert slot in the manual editor? I think it makes things cleaner